crates/router/src/core/payments/operations/payment_capture.rs

```diff
@@ -135,6 +128,46 @@ impl<F: Send + Clone> GetTracker<F, payments::PaymentData<F>, api::PaymentsCaptu
             .await
             .transpose()?;
 
+        let (capture, payment_attempt, connector_response) = match payment_attempt.capture_method {
+            Some(enums::CaptureMethod::ManualMultiple) => {
+                helpers::validate_attempt_status(payment_attempt.status)?;
+                let (capture, mut updated_payment_attempt) =
+                    Self::create_capture_and_update_attempt(state, payment_attempt, storage_scheme)
+                        .await
+                        .change_context(errors::ApiErrorResponse::InternalServerError)
+                        .attach_printable("Failed to create capture in DB")?;
+                let new_connector_response = db
+                    .insert_connector_response(
+                        Self::make_connector_response(&capture),
+                        storage_scheme,
+                    )
+                    .await
+                    .to_not_found_response(errors::ApiErrorResponse::PaymentNotFound)?;
+
+                //repopulate amount_to_capture field in updated attempt
+                updated_payment_attempt
+                    .amount_to_capture
+                    .update_value(request.amount_to_capture);
+                (
+                    Some(capture),
+                    updated_payment_attempt,
+                    new_connector_response,
+                )
+            }
+            _ => {
+                let connector_response = db
```

Comment:
When this case will occur? how manual multiple only need a capture and connector response?